content/swift/concepts/enums/enums.md

```diff
@@ -0,0 +1,151 @@
+---
+Title: 'Enums'
+Description: 'In Swift, an enumeration is a collection of set values that can be referenced throughout a program.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Enum'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+In Swift, an enumeration (enum) is a collection of set values that can be referenced throughout a program.
+
+## Syntax
+
+```swift
+enum MyEnum {
+  // Enumeration cases
+}
+```
+
+To access specific cases in `MyEnum`, the dot operator `.` is used, following the case name:
+
+```pseudo
+MyEnum.case
+```
+
+## Creating an `enum` and Examples
+
+`enum` cases can be defined on separate lines:
+
+```swift
+enum Level {
+  case beginner
+  case intermediate
+  case advanced
+}
+```
+
+`enum` cases can also be written in a single line for simplicity:
+
+```swift
+enum Level {
+  case beginner, intermediate, advanced
+}
+```
+
+### Raw Value
+
+Raw values can be added for each case if a raw type is defined after the `enum` name and trailing colon `:`.
```

Comment:
By "raw type" do we mean data types in Swift? If so, should we link "raw type" to this? 

https://www.codecademy.com/resources/docs/swift/data-types

Reply:
Yeah, good idea! I can link that